scripts/en2ja.json

```diff
@@ -0,0 +1,104 @@
+{
+  "title": "English-to-Japanese string replacement",
+  "description": "Replace EN string 'from' with associated JA string 'to'.",
+  "pairs": [
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/asa/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/asa/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/eu/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/eu/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oag/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oag/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oie/en-us",
```

Comment:
Searched for `https://help.okta.com/oie/en-us` in okta-help repo. There are some links we may be missing. Don't know if you have a separate jira for them. I'll just put them here in comment:
```
// many htm files reference en version from the content
https://github.com/okta/okta-help/blob/gh-pages/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/config-group-sync.htm#L316

// a bunch of files with REFRESH logic, i'm not sure how we use them
https://github.com/okta/okta-help/blob/gh-pages/en/programs/iga/Content/Topics/betas/closed/identity-governance/iga.htm

// sitemap
https://github.com/okta/okta-help/blob/gh-pages/en-us/Sitemap.xml
```

Reply:
There are also a few urls with relative paths like `../../../../asa/en-us/`, and I thought that we switched to relative paths in majority cases, that seems to be a wrong assumption.

Reply:
Within the `Content/Topics` directories of a given product, the URLs should be relative paths.
In the ASA update, I don't see any violators (grepping on `asa/en-us/`).
In the OIE baseline translation, I see seven to be exact. The first of these is doubly wrong, it's absolute and uses the Flare CSH function but not with an alias. This is a doc bug that a writer should fix in their content.
I'd handle this by doing an audit on the doc content in `info-dev`, and fixing these in source.
A global/product-wide search-and-replace of these in output wouldn't work as expected, because there are a few corner cases in which we want to have the hard-coded lang dir in the path.

Reply:
- links from content to other pages have to be fixed by writers ✅
- `http-equiv="REFRESH"` seems to be used only in betas ✅ 
- should we do anything with site map?

Reply:
Yes, thanks for bringing up sitemaps. Not covered here, created this ticket: [https://oktainc.atlassian.net/browse/OKTA-546296](https://oktainc.atlassian.net/browse/OKTA-546296)